src/bp/nlu/engine/typings.ts

```diff
@@ -1,3 +1,13 @@
+import {
+  IntentDefinition,
+  EntityDefinition,
+  PatternEntityDefinition,
+  ListEntityDefinition,
+  PredictOutput,
+  TrainInput
+} from '../typings_v1'
+
+export { IntentDefinition, EntityDefinition, PatternEntityDefinition, ListEntityDefinition, PredictOutput, TrainInput }
```

Comment:
mmmhhhh not a fan of re-exporting those types. There is so much typings inconsistency in NLU that we should at least make it clear on where the type was imported from.